### ISSUE TITLE
Select-only combobox Enter interaction diverges from native select

### ISSUE THREAD

**Issue by Mintoo200** [w3c/aria-practices#2998]
The [Select-only combobox example](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-select-only#kbd_label_combobox) states that pressing Enter on a closed combobox should "Opens the listbox without moving focus or changing selection."

The [combobox pattern page](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/#keyboardinteraction) does not state any specific interaction on a closed combobox when pressing Enter.

What is bothering me is that the native HTML select does nothing when the user presses Enter when the select is not associated to a form ([as demonstrated in this CodePen](https://codepen.io/Mintoo201/pen/yLrrxPz)) and [submits the form when associated to one](https://codepen.io/Mintoo201/pen/LYvvJQa) as you'd expect from any kind of input.

Is there a reason to expect Enter to open the listbox ?

Thanks !

**Comment by Mintoo200** [comment]
I realise this might be a question of how do you submit a form from a div. An [implementation with `requestSubmit`](https://codepen.io/Mintoo201/pen/LYvvgWV) might be an option ?

**Comment by juliebrunetto83** [comment]
Indeed, this is not the only interaction that differs from native select. Is there a reason for this?

**Comment by mcking65** [comment]
Per discussion in today's meeting, the "Enter" key will be added as an option for expanding select-only comboboxes to the combobox pattern.

**Comment by Mintoo200** [comment]
Hey, sorry to reopen this issue, just trying to understand. The select-only combobox does look like a menu button but it is used in a different context. This is basically an input type where, not only you can expand the dropdown list but, contrary to the menu, you need to keep track of the selected entry and handle it like you would a direct user input. It kinda feels weird not to expect the same features (in this case the whole "submit the form when pressing Enter").
Also, contrary to the menu button, this one will be announced as "combobox" to assistive technologies so the user might expect a different behaviour and different interactions.
Also also, given that both the native `<select>` and custom one will be announced as "combobox", the user wouldn't be aware that it is used in a different way, right?

Anyway, thanks for your feedback on this issue
